service/src/integrationTest/kotlin/fi/espoo/evaka/vasu/VasuIntegrationTest.kt

```diff
@@ -338,6 +341,15 @@ class VasuIntegrationTest : FullApplicationTest(resetDbBeforeEach = true) {
                 CurriculumType.PRESCHOOL -> ChildLanguage("kiina", "kiina")
             }
 
+        db.transaction { tx ->
```

Comment:
Should there be an assertion somewhere as well?

Reply:
Added!